src/main/java/org/folio/circulation/domain/notice/schedule/ScheduledDigitalReminderHandler.java

```diff
@@ -97,21 +112,48 @@ protected CompletableFuture<Result<ScheduledNoticeContext>> fetchLoan(
       .thenCompose(r -> r.after(loanPolicyRepository::findPolicyForLoan))
       .thenCompose(overdueFinePolicyRepository::findOverdueFinePolicyForLoan)
       .thenApply(mapResult(context::withLoan))
+      .thenApply(r -> {
+        String servicePointId = r.value().getLoan().getCheckoutServicePointId();
+        calendarRepository.lookupOpeningDays(ClockUtil.getZonedDateTime().toLocalDate(),servicePointId);
```

Comment:
It looks like we're not using the result of this lookup and it happens asynchronously. Do we actually need it?

Reply:
No, we don't. Good catch.